app/service.js

```diff
@@ -48,6 +51,12 @@ app.post('/v1/metadata', function(req, res) {
     return;
   }
 
+  const maxUrls = app.get('maxUrls');
+  if (req.body.urls.length > maxUrls) {
+    fail(errorMessages.maxUrls, 400);
```

Comment:
The current behaviours of the existing services are to reject the call entirely as invalid if the number of URLs requested exceeds the configured maximum:

https://github.com/mozilla/embedly-proxy/blob/master/embedly/embedly/tests/test_api.py#L103-L112